packages/api/src/graphql/mutation.private.ts

```diff
@@ -986,7 +869,7 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
       args: {
         value: {type: GraphQLList(GraphQLUpdateSettingArgs)}
       },
-      async resolve(root, {value}, {authenticate, dbAdapter, prisma}) {
+      async resolve(root, {value}, {authenticate, prisma}) {
```

Comment:
the updateSettingList business logic should also be outsourced like the other collections.

Reply:
Agree with you, came from merge with master. Will be done in this PR

Reply:
Done!